cmd/minikube/cmd/config/prompt.go

```diff
@@ -55,9 +56,14 @@ func AskForYesNoConfirmation(s string, posResponses, negResponses []string) bool
 }
 
 // AskForStaticValue asks for a single value to enter
-func AskForStaticValue(s string) string {
+func AskForStaticValue(s string, isOptional ...bool) string {
 	reader := bufio.NewReader(os.Stdin)
 
+	optional := false
+	if len(isOptional) > 0 {
```

Comment:
I understand that this one way to do it but this might be confusing and harder to maintain in the in the future.
Would it not be better to do something like:
```go
optional := false
if len(isOptional) > 0 {
	optional = isOptional[0]
}
```